pkg/kapp/cmd/app/migration_flags.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package app
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type MigrationFlags struct {
+	prevAppName string
+}
+
+func (s *MigrationFlags) Set(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&s.prevAppName, "prev-app", "", "Name of old app incase of migration")
```

Comment:
*Suggestion*: should we _not_ use the term migration for renaming? When we use the term migration I honestly feel that we are talking about the app suffix stuff and I get a bit confused 😅 
Maybe we can have:
```suggestion
	cmd.Flags().StringVar(&s.prevAppName, "prev-app", "", "Set previous name of app")
```
(Also, I know it's too late now but I feel that `--prev-name` might have been a good choice, it's similar to `--new-name` and also we would have used `prev-name` when `name` wasn't found vs `prev-app` when `name`  isn't found now.)

Reply:
P.S. We might also want to rename the file and the struct if we go ahead with the the above suggestion.

Reply:
the reason why i was proposing --prev-app as the flag name is because it's more consistent with the naming of --app.

```
  -a, --app string            Set app name (or label selector) (format: name, label:key=val, !key)
```

Reply:
Ah, I didn't think of it this way. 
Noice!